webapp/tests/karma/karma-unit.base.conf.js

```diff
@@ -5,15 +5,16 @@ module.exports = {
     require('karma-chrome-launcher'),
     require('karma-mocha-reporter'),
     require('@angular-devkit/build-angular/plugins/karma'),
-    require('karma-coverage'),
+    require('karma-coverage')
   ],
   client: {
-    captureConsole: true,
+    captureConsole: true
   },
   reporters: ['mocha', 'coverage'],
   mochaReporter: {
     output: 'full',
     showDiff: true,
+    suppressErrorSummary: true
```

Comment:
I think we want to see all error information in tests. Can you please remove this config?